rpi_backlight.py

```diff
@@ -144,9 +177,23 @@ def _create_argument_parser() -> argparse.ArgumentParser:
     )
     return parser
 
+def init() -> None:
+    global MODE, PATH
+    with open("/sys/firmware/devicetree/base/model") as f:
+        model_information = f.read()
+    if "Raspberry Pi" in model_information:
+        PATH = "/sys/class/backlight/rpi_backlight/"
+        MODE = "MODE_RPI"
+    elif "Tinker Board" in model_information:
+        PATH = "/sys/devices/platform/ff150000.i2c/i2c-3/3-0045/"
+        MODE = "MODE_TINKERBOARD"
+    else:
+        raise ValueError("unsupported OS, or OS could not be detected!")
+        sys.exit()
```

Comment:
Also this is still there

Reply:
What did you want this changed to?

Reply:
I thought I committed the changes you proposed for this one

Reply:
You said to remove the print and throw an exception

Reply:
I tried again please review

Reply:
Yep, I wanted the `sys.exit` call removed, looks good though. Wouldn't be reached after the exception anyways.